pyccel/codegen/printing/fcode.py

```diff
@@ -1089,12 +1291,53 @@ def _print_Assign(self, expr):
         if isinstance(rhs, (PythonRange, Product)):
             return ''
 
-        if isinstance(rhs, (NumpyArray, NumpyLinspace, NumpyDiag, NumpyCross,\
-						NumpyWhere, PyccelArraySize)):
+        if isinstance(rhs, (PythonLen, NumpyRandint)):
+            rhs_code = self._print(rhs)
+            return '{0} = {1}\n'.format(lhs_code, rhs_code)
+
+        if isinstance(rhs, PythonInt):
+            rhs = expr.rhs.fprint(self._print)
+            return '{0} = {1}\n'.format(lhs_code,rhs)
+        
+        if isinstance(rhs, (NumpyComplex, NumpyReal, NumpyArray, NumpyLinspace)):
+            rhs = self._print(rhs)
+            return '{0} = {1}\n'.format(lhs_code,rhs)
+        
+        if isinstance(rhs, (NumpyCross, NumpyDiag, NumpyWhere)):
+            return self._print(rhs, expr.lhs)
+        
+        if isinstance(rhs, PyccelArraySize):
             return rhs.fprint(self._print, expr.lhs) + '\n'
```

Comment:
Shouldn't this fprint be handled too?

Reply:
its not in ast.numpyext

Reply:
ok

Reply:
Do you know if there are many `fprint` functions outside of `ast.numpyext`? If there aren't very many then it might not be worth doing this in 2 PRs

Reply:
there some of them in builtins for python classes and their calls in fcode.

Reply:
With grep I see 4 methods. So I think it can be done in this PR